wandb/sdk/data_types/image.py

```diff
@@ -65,65 +65,72 @@ def _server_accepts_artifact_path() -> bool:
 class Image(BatchableMedia):
     """Format images for logging to W&B.
 
+    See https://pillow.readthedocs.io/en/stable/handbook/concepts.html#modes
+    for more information on modes.
+
     Args:
-        data_or_path: (numpy array, string, io) Accepts numpy array of
+        data_or_path: Accepts numpy array of
             image data, or a PIL image. The class attempts to infer
             the data format and converts it.
-        mode: (string) The PIL mode for an image. Most common are "L", "RGB",
-            "RGBA". Full explanation at https://pillow.readthedocs.io/en/stable/handbook/concepts.html#modes
-        caption: (string) Label for display of image.
+        mode: The PIL mode for an image. Most common are "L", "RGB",
+            "RGBA".
```

Comment:
this also fits in 80 characters

```suggestion
        mode: The PIL mode for an image. Most common are "L", "RGB", "RGBA".
```